src/components/Dialogs/PreferenceDialog/PreferenceDialogComponent.tsx

```diff
@@ -317,6 +317,47 @@ export class PreferenceDialogComponent extends React.Component {
                         onChange={(event: React.FormEvent<HTMLSelectElement>) => preference.setPreference(PreferenceKeys.WCS_OVERLAY_WCS_TYPE, event.currentTarget.value)}
                     />
                 </FormGroup>
+                <FormGroup inline={true} label="Colorbar Visible">
+                    <Switch checked={preference.colorbarVisible} onChange={ev => preference.setPreference(PreferenceKeys.WCS_OVERLAY_COLORBAR_VISIBLE, ev.currentTarget.checked)} />
+                </FormGroup>
+                <FormGroup inline={true} label="Colorbar Interactive">
+                    <Switch checked={preference.colorbarInteractive} onChange={ev => preference.setPreference(PreferenceKeys.WCS_OVERLAY_COLORBAR_INTERACTIVE, ev.currentTarget.checked)} />
+                </FormGroup>
+                <FormGroup inline={true} label="Position">
```

Comment:
To be consistent with the other labels, should this not be "Colorbar Position"?

Reply:
@veggiesaurus Yes, thanks. This is changed.